webservices/args.py

```diff
@@ -126,7 +126,9 @@ def make_seek_args(field=fields.Int, description=None):
 }
 
 query = {
-    'q': fields.Str(required=True, description='Text to search legal documents for')
+    'q': fields.Str(required=True, description='Text to search legal documents for'),
```

Comment:
Missing the period for consistency, although there is a lot of inconsistencies in the api docs https://api.open.fec.gov/developers/

+ @emileighoutlaw in case you want to review any of this copy.